src/components/02-structure/02-footer/footer.njk

```diff
@@ -10,8 +11,8 @@
         </span>
         <span class="sds-feedback__response-title-thin">
           Survey</span>
-        <p class="sds-feedback__response-text">What changes or improvements would you suggest?</p>
-        <input class="sds-feedback__input" type="text">
+        <label for="feedback" class="sds-feedback__response-text">What changes or improvements would you suggest?</label>
+        <textarea id="feedback" class="sds-feedback__input" type="text"></textarea>
```

Comment:
`type` attribute is unnecessary in textarea elements

Reply:
Done